charts/polkadot-base-services/templates/issuer.yaml

```diff
@@ -1,6 +1,6 @@
 {{ if .Values.createCertIssuer -}}
```

Comment:
It would be good to change the name of the file to `clusterissuer.yaml` wdyt?

Reply:
If we see an Issuer and a ClusterIssuer as a possibile implementation of something that is an Issuer, we should stick with the generic name issuer.yaml. wdyt? But I agree that clusterissuer.yaml is exactly what it is, so the choice is yours =)

Reply:
we are naming the template files following the resource kind, for consistency i think better move to `clusterissuer.yaml`